### PR TITLE
Angular 15

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Description

Update Angular packages

-----

- [x] Build
- [x] Storybook
- [x] Tests

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1910/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1910/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1910/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1910/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1910/storybook

**Review by mathieu-bellange — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -2,30 +2,40 @@
...
+    },
+    "declaration": false,
+    "downlevelIteration": true,
+    "experimentalDecorators": true,
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "target": "ES2022",
+    "module": "ESNext",
+    "lib": [
+      "es2020",
```
Comment (shajz) [inline tsconfig.json:34]:
```suggestion
      "ES2022",
```
?

**Review by shajz — APPROVED** [review]

**Pull request merged**